neoload-project/src/main/resources/as-code.latest.schema.json

```diff
@@ -64,14 +64,14 @@
 			"title": "Populations",
 			"type": "array",
 			"minItems": 1,
-			"items": { "$ref": "#/definitions/populations/population" }
+			"items": { "$ref": "#/definitions/population" }
 		},
 		"scenarios": {
 			"$id": "#root/scenarios",
 			"title": "Scenarios",
 			"type": "array",
 			"minItems": 1,
-			"items": { "$ref": "#/definitions/scenario" }
+			"items": { "$ref": "scenario.schema.json#/definitions/scenario-as-code" }
```

Comment:
I'm not sure about this part.
Is it changing the model?
If no I don't why it is not changing it.

Reply:
Doing this do not change the model.
I just say that i should retrieve the item definitions on the other file.

Renaming scenario to scenario-as-code should not change anything from a json point of view,